hw07/3.py

```diff
@@ -0,0 +1,25 @@
+# Описать функцию для перевода десятичного числа в
+# двоичное. Описать бесконечный цикл, в котором
+# запрашивать у пользователя число и переводить его в
+# двоичную систему. Признак окончания работы
+# программы - введенное пользователем число 0.
+# Подсказка: числа в двоичной системе хранить как
+# строки.
+
+def convert(num: int) -> str:
+    binary_num: str = ''
+    while num > 0:
+        binary_num += str(num % 2)
+        num = num // 2
+    return binary_num
+
+def main():
+    while True:
+        number = int(input())
+        if not number == 0 :
```

Comment:
Условие проверки на число всегда `True`,  если оно отлично от нуля. Так что это можно просто записать как:
```suggestion
        if number:
```

А вообще, в идеале, нужно стараться избегать использования `else` если это возможно. В твоем случае аккуратнее было бы так:
```suggestion
        if not number:
            return
        print(convert(number))
```